cypress/e2e/course/teacherCreateEditAndDeleteGroupInCourse.feature

```diff
@@ -0,0 +1,51 @@
+@release
+Feature: Teacher can create, edit and delete a group in the course
+
+@stable_test
+  Scenario Outline: Creating a group in course
+    Given I am logged in as a '<user>' at '<state>'
+    When I go to rooms overview
+    When I go to room '<course_title>'
+    When I click on tab Groups
+    Then I see button Create a new group
+    When I click on button Create a new group
+    When I see page Create student group
+    When I type '<group_name>' in field Group name
+    When I select '<group_member>' from field Group member
+    When I click on button Create student group
+    Then I see group is created with name '<group_name>'
+    Examples:
+            |   user         |    state  |    course_title   |    group_name  |    group_member        |
+            |   teacher1_brb |    brb    |    German         |    Group-Work  |    Amelia Strobl      |
+
+@stable_test
+  Scenario Outline: Editing a group in course
+    Given I am logged in as a '<user>' at '<state>'
```

Comment:
you can remove this log in step from edit and delete scenarios as it makes tests slow. we are using session which helps in eliminating login multiple times